### PR TITLE
Issue 52

### LINKED ISSUE #52: First pass at normalize endpoint
1. Get variation from `/toVRS`
2.  Build a [Value Object Descriptor](https://vrsatile.readthedocs.io/en/latest/value_object_descriptor/vod_index.html#value-object-descriptor)

### PR DESCRIPTION
**Pull request by korikuzma**
Closes #52

### CODE REVIEW

File: variant/schemas/ga4gh_vod.py [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+    value_id: Optional[str]
+    value: Optional[dict]
+    xrefs: Optional[List[str]]
+    alternate_labels: Optional[List[str]]
+    extensions: Optional[List[Extension]]
+
+    class Config:
+        """Configure model."""
+
+        orm_mode = True
```
Comment (jsstevenson) [inline variant/schemas/ga4gh_vod.py:45]: do these need ORM mode? (genuine question, I don't remember why we had to turn this on)

**Review by jsstevenson — APPROVED** [review]
👍

File: variant/schemas/ga4gh_vod.py (hunk 1 above)
Comment (korikuzma) [reply variant/schemas/ga4gh_vod.py:45]: Removed!

File: variant/normalize.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+            record_location = record.locations[0] if record.locations else None
+
+            return GeneDescriptor(
+                id=f"normalize:{gene_symbol}",
+                label=gene_symbol,
+                value=Gene(gene_id=record.concept_id),
+                xrefs=record.other_identifiers,
+                alternate_labels=[record.label] + record.aliases + record.previous_symbols,  # noqa: E501
+                extensions=self.get_extensions(record, record_location)
+            )
```
[the remark this answers is not in the document]
Comment (korikuzma) [reply variant/normalize.py:86]: Added!

File: variant/normalize.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+            None.
+        """
+        if len(validations.valid_results) > 0:
+            # For now, only use first valid result
+            valid_result = validations.valid_results[0]
+            valid_result_tokens = \
+                validations.valid_results[0].classification.all_tokens
+
+            polypeptide_sequence_variant_token = \
+                ([t for t in valid_result_tokens if isinstance(t, PolypeptideSequenceVariant)] or [None])[0]  # noqa: E501
```
[the remark this answers is not in the document]
Comment (korikuzma) [reply variant/normalize.py:30]: Ope! Forgot to change. Thanks!

**Pull request merged**